test/src/requestinterception.spec.ts

```diff
@@ -144,7 +144,7 @@ describe('request interception', function () {
         }
         const headers = request.headers();
         headers['test'] = 'test';
-        void request.continue({headers});
+        void request.continue({headers: request.headers()});
```

Comment:
I think this is using `headers` on purpose.

Reply:
was this test affected by the changes here?

Reply:
Yes the change affects this test, as the test is sending new Headers to be send along side the request. 
I think the test intention was to make sure the object is writable.